templates/output.html.twig

```diff
@@ -2,46 +2,56 @@
 
 {% block body %}
     <form method="get">
-        <div class="form-group">
-            <label for="image">{{ msg('image-url') }}</label>
-            <input name="image" id="image" class="form-control" type="url" value="{{ app.request.get('image') }}" required />
-            <p class="help-block">{{ msg( 'image-url-help', [image_hosts] ) }}</p>
-        </div>
-        <div class="form-group">
-            <label for="lang">{{ msg('language-code') }}</label>
-            <select id="lang" class="form-control" name="langs[]" multiple>
-                {% for lang in available_langs %}
-                    <option value="{{ lang }}" {% if lang in langs %}selected{% endif %}>
-                        {{- lang }}{% if ocr_lang_name(lang) is not empty %} &ndash; {{ ocr_lang_name(lang) }}{% endif -%}
-                    </option>
-                {% endfor %}
-            </select>
-            <p class="help-block">{{ msg( 'language-code-help' ) }}</p>
-        </div>
-        <div class="form-group">
-            <label for="engine">{{ msg('engine') }}</label>
-            <select name="engine" id="engine" class="form-control">
-                <option value="google" {% if engine == 'google' %}selected{% endif %}>{{ msg('engine-name-google') }}</option>
-                <option value="tesseract" {% if engine == 'tesseract' %}selected{% endif %}>{{ msg('engine-name-tesseract') }}</option>
-            </select>
-            <p class="help-block">{{ msg( 'engine-help' ) }}</p>
-        </div>
+        <fieldset>
+            <legend>{{ msg('form-heading') }}</legend>
+            <div class="form-group">
+                <label for="image">{{ msg('image-url') }}</label>
+                <input name="image" id="image" class="form-control" type="url" value="{{ app.request.get('image') }}" required />
+                <p class="help-block">{{ msg( 'image-url-help', [image_hosts] ) }}</p>
+            </div>
+            <div class="form-group">
+                <label for="engine">{{ msg('engine') }}</label>
```

Comment:
This is invalid HTML, since the "for" attribute no longer has a valid element ID. Since both radios already have labels, this "for" attribute can just be removed.

Reply:
Done!